packages/amplify_analytics_plugin_interface/lib/amplify_analytics_plugin_interface.dart

```diff
@@ -63,4 +63,8 @@ abstract class AnalyticsPluginInterface extends AmplifyPluginInterface {
       required AnalyticsUserProfile userProfile}) async {
     throw UnimplementedError('identifyUser() has not been implemented.');
   }
+
+  // Internal
+
+  Future<void> onConfigure() async {}
```

Comment:
Would that be better to convert the `// internal` to the method document to explicitly call out that this API is internally used by amplify-flutter library?

Reply:
Marked as protected